.github/workflows/clippy.yml

```diff
@@ -42,7 +43,7 @@ jobs:
         uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --manifest-path ./bindings/wasm/Cargo.toml --all-targets --all-features -- -D warnings
+          args: --manifest-path ./bindings/wasm/Cargo.toml --target wasm32-unknown-unknown --all-targets --all-features -- -D warnings
```

Comment:
If I run this command without `--target` from the root, I get tons of errors because it checks the Wasm bindings with target `x86_64-unknown-linux-gnu` (in my case). But we never compile the Wasm bindings for that target, so it's throwing errors we don't care about. In fact, I think this was always wrong, and when I first ran this with the wasm target, I got a clippy lint (`drop_copy`, which is why `Copy` was removed on `WasmTimestamp`) that wasn't triggered under the non-Wasm targets. Note that `--all-targets` refers to cargo targets, not target architectures. Maybe that's where the misunderstanding comes from? Until recently, I also interpreted that incorrectly.

Reply:
Interesting, I wasn't aware of the errors since I'm compiling on Windows. I was definitely under the impression that it was target architectures (which doesn't make sense in hindsight) rather than Cargo targets lib, examples, test etc.

Specifying `--target wasm32-unknown-unknown` makes perfect sense then.